ProjectReunion.sln

```diff
@@ -115,8 +115,8 @@ Global
 		{C62688A1-16A0-4729-B6ED-842F4FAA29F3}.Release|x64.Build.0 = Release|x64
 		{C62688A1-16A0-4729-B6ED-842F4FAA29F3}.Release|x86.ActiveCfg = Release|Win32
 		{C62688A1-16A0-4729-B6ED-842F4FAA29F3}.Release|x86.Build.0 = Release|Win32
-		{7C502995-59C3-483B-86BA-815985353633}.Debug_test|Any CPU.ActiveCfg = Release|x64
```

Comment:
>Debug_test [](start = 41, length = 10)

What's the Debug_test Configuration used for? Do any *.vcxproj reference it? Artifact of initial solution creation or is there a need?

Any reason we can't consolidate on Debug?

Reply:
The SLN file is generated, and these changes are from VS.  Specifically me fixing one of the flavors.  It sounds like your question should be for Elliot and/or Ben and unrelated to my changes.